version.c

```diff
@@ -176,6 +190,14 @@ define_ruby_description(const char *const jit_opt)
     RUBY_ASSERT(n <= ruby_description_opt_point + (int)rb_strlen_lit(YJIT_DESCRIPTION));
     if (ruby_mn_threads_enabled) append(" +MN");
     if (rb_ruby_prism_p()) append(" +PRISM");
+#if USE_SHARED_GC
+    append(GC_DESCRIPTION);
+    if (getenv("RUBY_GC_LIBRARY")) {
+        append("[");
+        append(rb_gc_active_gc_name());
```

Comment:
Since `rb_gc_active_gc_name` returns a malloc'd buffer, I think this will leak memory.